internal/codegen/tpl_file.go

```diff
@@ -72,11 +69,11 @@ func (f *TplFile) SetContents(contents string) error {
 
 // Skip skips the current file being rendered
 //
-//	{{ $_ := file.Skip "A reason to skip this reason" }}
-func (f *TplFile) Skip(reason string) error {
+//	{{ file.Skip "A reason to skip this file" }}
```

Comment:
```suggestion
//	{{- file.Skip "A reason to skip this file" }}
```

Probably good to have examples show the `{{-` so that the act of calling `file.Skip` doesn't return an empty new line (default behaviour without the leading `-`)

Reply:
Likely want to repeat this for all examples as well (thank you!)